recipes/mykrobe/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: mykrobe
+  version: "0.5.6"
+
+source:
+  fn: mykrobe-0.5.6.tar.gz
+  url: https://pypi.python.org/packages/7e/35/a662ec560cfc91f35519e1459de917001ca19a9dbefd53c5b3a2b14122d0/mykrobe-0.5.6.tar.gz
+  md5: 5e912497f98cc192e3f0fbf426210560
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - mykatlas
+    - ga4ghmongo
+    - pyvcf
+    - mongoengine
+    - biopython
+    - future
+
+  run:
+    - python
+    - mykatlas
+    - ga4ghmongo
+    
+test:
+  # Python imports
+  imports:
+    - mykrobe
+  commands:
+    - mykrobe --help
+
+
+about:
+  home: https://github.com/iqbal-lab/Mykrobe-predictor
+  license: https://github.com/iqbal-lab/Mykrobe-predictor/blob/master/LICENSE
```

Comment:
There is a special `license_file` field we can use that takes a filename from the source archive:
```yaml
  license: Custom non-commercial license
  license_file: LICENSE
```
(The project's license also requires us to include it in the package this way.)

Reply:
Since using pipy, the license_file is not part of the source  url:

Reply:
Fixed